src/types.rs

```diff
@@ -640,6 +647,8 @@ pub struct FieldInfo {
     pub bitfields: Option<Vec<(String, u32)>>,
     /// If the C++ field is marked as `mutable`
     pub mutable: bool,
+    pub private: bool,
```

Comment:
Add a doc comment here saying when this can be true (right now only explicitly via annotations right?)